lib/packExternalModules.js

```diff
@@ -182,36 +182,42 @@ function isExternalModule(module) {
   return _.startsWith(module.identifier(), 'external ') && !isBuiltinModule(getExternalModuleName(module));
 }
 
+/**
+ * Gets the module issuer. The ModuleGraph api does not exists in webpack@4
+ * so falls back to using module.issuer.
+ */
+function getIssuerCompat(moduleGraph, module) {
+  if (moduleGraph != null) {
```

Comment:
It should be enough, don't you think?

```suggestion
  if (moduleGraph) {
```